iceoryx_posh/test/integrationtests/test_shm_sigbus_handler.cpp

```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// SPDX-License-Identifier: Apache-2.0#include "test.hpp"
+// SPDX-License-Identifier: Apache-2.0
 
 #if !defined(__APPLE__)
-
+#include "test.hpp"
```

Comment:
Oh, ``test.hpp`` is already included in l. 24... oops

Reply:
My bad, i didn't saw it